src/Sql/Sql/TransparentDataEncryption/Model/ManagedInstanceKeyHelper.cs

```diff
@@ -0,0 +1,61 @@
+﻿// ----------------------------------------------------------------------------------
+//
+// Copyright Microsoft Corporation
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+// http://www.apache.org/licenses/LICENSE-2.0
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+// ----------------------------------------------------------------------------------
+
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Text.RegularExpressions;
+using System.Threading.Tasks;
+
+namespace Microsoft.Azure.Commands.Sql.TransparentDataEncryption.Model
+{
+    /// <summary>
+    /// Helper class for Managed instance key
+    /// </summary>
+    class ManagedInstanceKeyHelper
```

Comment:
Isn't this all the same for server key? I think you don't need to duplicate it. Just rename to KeyHelper.

Reply:
addressed